src/main/resources/static/css/techpoker.css

```diff
@@ -1,20 +1,20 @@
 .poker-card {
     /* Add shadows to create the "card" effect */
-    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+    box-shadow: 0 4px 8px 0 rgba(255, 255, 255, 0.1);
     transition: 0.3s;
     border-radius: 10px; /* 5px rounded corners */
-    min-height: 16em;
```

Comment:
This was mostly so the button looked like cards, now that the scaling issue seems to have been fixed on mobile, we can add this back in, I think.

Reply:
Done